packages/ui-core/src/components/modal/modal.component.tsx

```diff
@@ -47,18 +51,46 @@ export const Modal: FC<Props> = ({
   }, [closeable]);
 
   useEffect(() => {
+    const preventDefault = (e: MouseEvent) => {
+      e.preventDefault();
+    };
+
+    const keysArray = [33, 34, 38, 40];
+
     const keyboardHandler = (keyEvent: KeyboardEvent) => {
```

Comment:
You could use `useCallback` from `React` to avoid creating a new `keyboardHandler` function on each component re-render and move handler outside useEffect hook.